backup.js

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env node
+
+const { exec } = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const arg = require('arg');
+
+/* eslint-disable no-console no-process-env */
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Definition for rule 'no-console no-process-env' was not found.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5305080)